src/lib/holocene/combobox/combobox.svelte

```diff
@@ -422,133 +422,134 @@
-        type="text"
-        value={displayValue}
-        class:disabled
+      <div
         class={merge(
-          'combobox-input',
-          multiselect
-            ? value.length > 0 || leadingIcon
-              ? 'indent-0'
-              : 'indent-1'
-            : 'indent-2',
-          className,
+          'input-wrapper',
+          multiselect && 'gap-1',
+          multiselect && 'm-1',
+          leadingIcon && multiselect && 'ml-2',
         )}
-        role="combobox"
-        autocomplete="off"
-        autocapitalize="off"
-        spellcheck="false"
-        data-lpignore="true"
-        data-1p-ignore="true"
-        aria-controls="{id}-listbox"
-        aria-expanded={$open}
-        aria-required={required}
-        aria-autocomplete="list"
-        on:focus|stopPropagation={openList}
-        on:input|stopPropagation={handleInput}
-        on:keydown|stopPropagation={handleInputKeydown}
-        on:click|stopPropagation={handleInputClick}
-        data-testid={$$props['data-testid'] ?? id}
-        bind:this={inputElement}
-        {...$$restProps}
-      />
-    </div>
-    {#if $$slots.action}
-      <div class="ml-1 flex h-full items-start border-l border-subtle p-0.5">
-        {#if actionTooltip}
-          <Tooltip text={actionTooltip} right>
-            <slot name="action" />
-          </Tooltip>
-        {:else}
-          <slot name="action" />
+      >
+        {#if multiselect && isArrayValue(value) && value.length > 0}
+          {#if displayChips}
+            {#each value.slice(0, chipLimit) as v}
+              <Chip
+                on:remove={() => removeOption(v)}
+                removeButtonLabel={removeChipLabel}>{v}</Chip
+              >
+            {/each}
+            {#if value.length > chipLimit}
+              <p>+{value.slice(chipLimit).length}</p>
+            {/if}
+          {:else}
+            <Badge>{numberOfItemsSelectedLabel(value.length)}</Badge>
+          {/if}
         {/if}
+        <input
+          {id}
+          {placeholder}
+          {required}
+          {readonly}
+          {disabled}
+          type="text"
+          value={displayValue}
+          class:disabled
+          class={merge(
+            'combobox-input',
+            multiselect
+              ? value.length > 0 || leadingIcon
+                ? 'indent-0'
+                : 'indent-1'
+              : 'indent-2',
+            className,
+          )}
+          role="combobox"
+          autocomplete="off"
+          autocapitalize="off"
+          spellcheck="false"
+          data-lpignore="true"
+          data-1p-ignore="true"
+          aria-controls="{id}-listbox"
+          aria-expanded={$open}
+          aria-required={required}
+          aria-autocomplete="list"
+          on:focus|stopPropagation={openList}
+          on:input|stopPropagation={handleInput}
+          on:keydown|stopPropagation={handleInputKeydown}
+          on:click|stopPropagation={handleInputClick}
```

Comment:
I don't like seeing stopPropagations do we have a good reason for these? Asking because you have context you don't have to remove it just trying to figure out why we have it all over

Reply:
Probably because we don't want to submit any form on the page current viewing. Like if you on schedules create, stop propagation of form